gcam-usa.md

```diff
@@ -44,6 +44,8 @@ Overall, despite growing electricity demand, the assumption of no new freshwater
 
 Water supply: Our reference scenario adopts the renewable and non-renewable water supplies that are calculated and implemented for all water basins in the USA region in the global GCAM. 
 
+Emissions: Future air pollutant emissions assumptions are implemented to reflect current rules and legislation. This approach results in a near-term reduction in emissions for the sectors that currently have regulations applied to them. The specific regulations by sector are listed below (Table 2). Further into the future, around 2050 on, some emissions begin to increase again. This happens in sectors with minimal legislation to continue reducing emissions, and where no GDP control is applied. Without the addition of legislation and/or a GDP control, emissions continue to increase as their drivers (whether population or energy consumption) also increase. 

```

Comment:
Thanks for adding this Maridee!  One note, I think we need to define GDP control the first time we use it here, or at the very least link to where it is defined in the emissions documentation.  I don't think this is necessarily common terminology for public users.